src/interface/metadata.hpp

```diff
@@ -548,7 +552,6 @@ class Metadata {
 
   // Refinement stuff
   const refinement::RefinementFunctions_t &GetRefinementFunctions() const {
-    PARTHENON_REQUIRE_THROWS(IsRefined(), "Variable must be registered for refinement");
```

Comment:
We are calling `GetRefinementFunctions()` here: 

https://github.com/parthenon-hpc-lab/parthenon/pull/992/files#diff-e142fce743a6bbfa5d718f415ca450fd5abf0694750a16562c64f7b5e392d5e6R62

With the current design, we need to make this call even for fields that are *not* `IsRefined()`.  In this scenario, we are passing an unset `refinement_funcs_`, which is never actually used because there is an internal check on `IsRefined()` inside the `Metadata` constructor, followed by an additional `IsValid()`.

Reply:
Oh I see. So long as it's still safe somewhere, I think this is fine. Tests will fail though (because I think this is a test case.)